sys/net/gnrc/network_layer/sixlowpan/nd/gnrc_sixlowpan_nd.c

```diff
@@ -154,10 +154,11 @@ kernel_pid_t gnrc_sixlowpan_nd_next_hop_l2addr(uint8_t *l2addr, uint8_t *l2addr_
     if (nc_entry != NULL) {
         gnrc_ipv6_netif_t *ipv6_if = gnrc_ipv6_netif_get(nc_entry->iface);
         /* and interface is not 6LoWPAN */
-        if (!(ipv6_if->flags & GNRC_IPV6_NETIF_FLAGS_SIXLOWPAN) ||
+        if (!((ipv6_if == NULL) ||
```

Comment:
The function may be called with the `nc_entry->iface` is `0`, resulting in a null pointer dereference. See commit message for detail.
https://github.com/Yonezawa-T2/RIOT/commit/b0e9c406a1b7ff8aff37f99d8d5a2b4d0117542f

There would be a better comprehensive solution for coordinating NDPs for conventional IPv6 and 6LoWPAN, but it would require a large restructuring, so that ad hoc workaround here.